data/strings/strings.txt

```diff
@@ -28803,3 +28803,48 @@
     vi = Âm lượng
     zh-Hans = 音量
     zh-Hant = 音量
+
+  [bookmarks_export]
+    tags = android
+    comment = Bookmark categories screen, button that opens folder selection dialog to export KML/KMZ/GPX/KMB files
+    en = Export Bookmarks and Tracks
+    af = Voer boekmerke en snitte uit
+    ar = تصدير الإشارات المرجعية والمسارات
+    az = Əlfəcinləri və Parçaları ixrac edin
+    be = Экспартаваць закладкі і трэкі
+    bg = Експортиране на отметки и следи
+    ca = Exporta adreces d'interès i pistes
+    cs = Export záložek a stop
+    da = Eksport af bogmærker og spor
+    de = Lesezeichen und Tracks exportieren
+    el = Εξαγωγή σελιδοδεικτών και διαδρομών
+    es = Exportar marcadores y pistas
+    et = Ekspordi järjehoidjad ja lood
+    eu = Esportatu laster-markak eta ibilbideak
+    fa = صادرات نشانک ها و آهنگ ها
+    fi = Kirjanmerkkien ja raitojen vienti
+    fr = Exporter les signets et les pistes
+    he = ייצוא סימניות ורצועות
+    hi = बुकमार्क और ट्रैक निर्यात करें
+    hu = Könyvjelzők és sávok exportálása
```

Comment:
```suggestion
    hu = Könyvjelzők és útvonalak exportálása
```

Reply:
Thanks for the suggestion, english text was slightly changed, could you please review updated translation